packages/react/src/TreeView/TreeView.tsx

```diff
@@ -200,11 +210,16 @@ const UlBox = styled.ul<SxProp>`
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
+    align-self: center;
```

Comment:
If we remove the flex start and line height above, I don't think we would need this. Let me know if I am missing any context here though!